components/InstructionsAccordion.tsx

```
@@ -0,0 +1,68 @@
+      <Accordion
+        type="single"
+        collapsible
+        className="w-full"
+        value={currentStep.toString()}
+      >
+        {StepDataSet.map((stepData) => {
+          const isActive = stepData.step === currentStep;
+          const progressValue = isActive ? stepProgress : 0;
+
+          return (
+            <AccordionItem value={stepData.step.toString()}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)